fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBMetaDataStore.java

```diff
@@ -420,6 +425,89 @@ public void saveRecordMetaData(@Nonnull RecordMetaDataProto.MetaData metaDataPro
         context.asyncToSync(FDBStoreTimer.Waits.WAIT_SAVE_META_DATA, saveAndSetCurrent(metaDataProto));
     }
 
+    private CompletableFuture<RecordMetaDataProto.MetaData> loadCurrentProto() {
```

Comment:
I filed an issue for this (#409), but now that I look into it closer, I'm not sure if changing this would buy us much? Both `loadAndSetCurrent` and `saveAndSetCurrent` require the serialized proto that they load. Unless you feel strongly about it, how about I close that issue?

Reply:
Nah, I suppose it's fine. I might leave a comment in #409 when you close it about why it can't/shouldn't be done.